### DISCUSSION TITLE
Does mongoose support mongo stable api?

### DISCUSSION THREAD

**Discussion by KKHKO** [Automattic/mongoose#12875] [Q&A] [answered]
Does mongoose support mongo stable api?

My code:
mongoose.connect(url, { useNewUrlParser: true , autoIndex: false })

I know MongoDB node driver support mongo stable api : 
client = new MongoClient(uri, { serverApi: { version: '1', strict: false } });
(https://www.mongodb.com/docs/manual/reference/stable-api/#migrate-to-stable-api-commands)

**Comment by vkarpov15** [comment, marked as answer]
You should be able to do `mongoose.connect(url, { serverApi: { version: '1', strict: false }, autoIndex: false })`

Just make sure you're using Mongoose 6 or higher. Mongoose 5 doesn't support the stable API AFAIK, and `useNewUrlParser` is only supported in Mongoose 5, which makes me think you're on an old version of Mongoose.

**Reply by KKHKO** [reply]
thx for your help!
